README.md

```diff
@@ -328,7 +333,7 @@ In the following interactions, can you spot which guideline applies?
 > It’s a polyfill used in lazyLoad.js
 >
 > Jamie:
-> Yes I got that, but why not importing there instead of here? I know that the import is not executed always in lazyload.js, but I see it weird.. Maybe a require() instead?
```

Comment:
Though your suggestions are correct, I'd prefer to leave these examples exactly as I found them in real code reviews. The idea of this section is to find which guidelines would apply to them. It looks you've found quite some!